AdvanceTaskMarsPart1/Pages/ProfilePage.cs

```diff
@@ -0,0 +1,80 @@
+﻿using AdvanceTaskMarsPart1.Utilities;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace AdvanceTaskMarsPart1.Pages
+{
+    public class ProfilePage : CommonDriver
```

Comment:
So same comments apply to all pages around breaking down page into smaller components